helmfile/values/prometheus-blackbox-exporter-wc.yaml.gotmpl

```diff
@@ -67,32 +67,36 @@ serviceMonitor:
       module: http_2xx
     {{- end }}
 
+    {{- if .Values.prometheusBlackboxExporter.targets.sc}}
+    {{- $scDomain := default .Values.global.baseDomain .Values.global.scDomain }}
+    {{- $scOpsDomain := default .Values.global.opsDomain .Values.global.scOpsDomain }}
```

Comment:
Is it me looking in the wrong place or is `default` missing from the [helmfile docs](https://helmfile.readthedocs.io/en/latest/templating_funcs/)?

Reply:
I looked at how it was done [here](https://github.com/elastisys/compliantkubernetes-apps/blob/de2875d3467ea5008eca9e43d605d57fb06219c2/helmfile/values/thanos/objectstorage-secret.yaml.gotmpl#L8) and tested it out and it worked. Maybe it needs to be documented, and I am open for suggestions if there is a better or more clear way to do this.

Reply:
Best case, someone forgot to document it. Worst case, it's some unofficial feature that might change behavior in a future version without notice.

Actually it seems to be an omission on that template functions page, [this page does mention it](https://helmfile.readthedocs.io/en/latest/writing-helmfile/#missing-keys-and-default-values).

Reply:
It should just be go templating (and sprig templates) under the hood, and there is docs for the default function [here](http://masterminds.github.io/sprig/defaults.html)